src/client/user/components/CreateUrlModal/CreateLinkForm.tsx

```diff
@@ -300,6 +307,76 @@ const CreateLinkForm: FunctionComponent<CreateLinkFormProps> = ({
               </a>
             </CollapsibleMessage>
           </div>
+          <div className={classes.labelText}>
+            <Typography variant="body1">
+              Tag (add up to <strong>{MAX_NUM_TAGS_PER_LINK} tags</strong>){' '}
+              <Tooltip
+                title="Tags are words, or combinations of words, you can use to classify or describe your link."
+                imageAltText="Tags help"
+              />
+            </Typography>
+          </div>
+          <div>
+            <Autocomplete
```

Comment:
Might want to check these warnings while filling up the Autocomplete text field:

![image](https://user-images.githubusercontent.com/65295129/189035732-d28acd26-4b55-42bb-8d6e-50d33faee94c.png)

Reply:
thanks for the catch, i'm adding [`freeSolo`](https://v4.mui.com/components/autocomplete/#free-solo) to Autocomplete to stop this from happening!